scripts/ci-e2e.sh

```diff
@@ -48,7 +48,9 @@ get_random_region() {
     echo "${REGIONS[${RANDOM} % ${#REGIONS[@]}]}"
 }
 
-export REGISTRY="e2e"
+export TIMESTAMP="$(date -u '+%Y-%m-%dT%H:%M:%SZ')"
```

Comment:
TIMESTAMP is already exported below on line 75

https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/852/files#diff-20e0541def8acb609f41e6e8e278ae08L73